src/main/java/bio/terra/cli/businessobject/Context.java

```diff
@@ -117,22 +118,28 @@ public static void synchronizeToDisk() {
    */
   public static Path getContextDir() {
     // default to the user's home directory
-    Path parentDir = Paths.get(System.getProperty("user.home"));
-
-    // if the override environment variable is set and points to a valid directory, then use it
-    // instead
+    String contextDir = System.getProperty("user.home");
+    // if the override environment variable is set, use it instead
     String overrideDirName = System.getenv(CONTEXT_DIR_OVERRIDE_NAME);
     if (overrideDirName != null && !overrideDirName.isBlank()) {
-      Path overrideDir = Paths.get(overrideDirName).toAbsolutePath();
-      if (overrideDir.toFile().exists() && overrideDir.toFile().isDirectory()) {
```

Comment:
we already needed to mkdir for the worker-specific directories, switching the behavior for a non-existent `CONTEXT_DIR_OVERRIDE_NAME` from an exception to making the dir was mostly just convenient. 
It also only affects cases where the user explicitly specified `-PcontextDir`, since otherwise we would run `mkdir` from `build.gradle`